Scripts/Runtime/Entities/TaskSystem/TaskData/DataStream/Profiling/DataStreamProfilingSystem.cs

```diff
@@ -0,0 +1,60 @@
+using Anvil.Unity.DOTS.Jobs;
+using System.Collections.Generic;
+using Unity.Entities;
+
+namespace Anvil.Unity.DOTS.Entities.Tasks
+{
+#if DEBUG
+    /// <summary>
+    /// System that will handle populating profiling counters once per frame when guaranteed that all writing jobs
+    /// have completed.
+    /// </summary>
+    //TODO: #86 - Revisit with Entities 1.0 for "Create Before/After"
+    //TODO: #108 - Revisit to see if there is a better way to handle acquire/release. Or can we do it all in a job? 
+    [UpdateInGroup(typeof(PresentationSystemGroup), OrderLast = true)]
```

Comment:
We don't have this group in the simulation worlds right now. We should probably have this happen at the very start of the initialization group or end of the simulation group.

I do have thoughts around introducing some new phases into the PlayerLoop for Anvil but I'm not sure how soon those will be introduced yet and if they're a good idea.